bin/all_sky_search/pycbc_template_kde_calc

```diff
@@ -0,0 +1,147 @@
+#3dimensional_kde 
+#python3 pycbc_template_kde_calc.py --template-file /home/praveen.kumar/focused_search/kde/sbbh2.hdf --signal-file /home/praveen.kumar/focused_search/kde/3OGC_PEtable.txt_ifar_gt0.5.csv     --make-signal-kde    --verbose
+
+import numpy as np
+import operator
+import pycbc
+import h5py
+import pycbc.conversions as convert
+from utils_awkde import *
+from awkde import GaussianKDE
+from sklearn.model_selection import KFold
+import argparse
+import logging
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument('--signal-file', required=True)
+parser.add_argument('--template-file', required=True)
+parser.add_argument('--which-kde', choices=['signal','template'])
```

Comment:
I don't think this is used in the code at the moment, so can you remove this option _or_ use it to switch between the two cases?